src/OpenLoco/CMakeLists.txt

```diff
@@ -326,49 +326,49 @@ set(OLOCO_SOURCES
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/MapWindow.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/MessageWindow.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Map.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/MapTooltip.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Messages.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/MusicSelection.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/NetworkStatus.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/News/Common.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/News/News.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/News/Ticker.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ObjectLoadError.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ObjectSelectionWindow.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ObjectSelection.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Options.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PlayerInfoPanel.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ProgressBar.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptBrowseWindow.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptOkCancelWindow.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptSaveWindow.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptBrowse.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptOkCancel.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/PromptSave.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ScenarioOptions.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ScenarioSelect.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Station.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/StationList.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/StationWindow.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TerraForm.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TextInputWindow.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Terraform.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TextInput.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TileInspector.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TimePanel.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TitleExit.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TitleLogo.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TitleMenu.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TitleOptions.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/TitleVersion.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ToolTip.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ToolbarBottomEditor.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ToolbarTop.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ToolbarTopAlt.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/ToolbarTopCommon.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/Ui/Windows/Toolbar/Common.cpp"
```

Comment:
this should all be alphabetical and grouped by folder so the /Toolbar/ should all be grouped together at the start of Windows/